r/README.Rmd

```diff
@@ -29,12 +29,26 @@ cmake .. -DCMAKE_BUILD_TYPE=Release -DARROW_BOOST_USE_SHARED:BOOL=Off
 make install
 ```
 
-Then the R package: 
+Then the R package:
 
 ```r
 devtools::install_github("apache/arrow/r")
```

Comment:
Is this the preferred workflow for development? I haven't tried building the package yet but am curious about the iteration cycle for developers